src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -964,6 +964,12 @@ public SyntaxList<SyntaxNode> GetMembersOfNamespaceDeclaration(SyntaxNode namesp
         public SyntaxList<SyntaxNode> GetMembersOfCompilationUnit(SyntaxNode compilationUnit)
             => ((CompilationUnitSyntax)compilationUnit).Members;
 
+        public SyntaxList<SyntaxNode> GetImportsOfNamespaceDeclaration(SyntaxNode namespaceDeclaration)
+            => ((NamespaceDeclarationSyntax)namespaceDeclaration).Usings;
```

Comment:
```suggestion
            => ((BaseNamespaceDeclarationSyntax)namespaceDeclaration).Usings;
```

also add test for filescopednamespace case.

Reply:
(note: you'll need to fixup IsNamespaceDeclaration.  i'm doing that in one of my PRs.  but it's ok if you do that as well.